### PR TITLE
Added basic help menu

### PR DESCRIPTION
**Pull request by Tangdongle**
Refactored setForegroundColor calls
Fixed up indenting from previous commits

### CODE REVIEW

File: src/inim.nim [hunk 1]
```diff
@@ -248,10 +247,24 @@ proc runForever() =
...
 
+        if currentExpression in ["help", "help()"]:
+            outputFg(fgCyan, true):
+                echo("""
+                iNim - Interactive Nim Shell - By AndreiRegiani
+
+                Available Commands:
+                Quit - exit, exit(), quit, quit()
+                Help - help, help()
+                """
```
Comment (0atman) [inline src/inim.nim:263]: I think you need to de-indent this string, it looks like this when you print it:

```
nim> help 
                iNim - Interactive Nim Shell - By AndreiRegiani

                Available Commands:
                Quit - exit, exit(), quit, quit()
                Help - help, help()
```

try https://nim-lang.org/docs/strutils.html#unindent%2Cstring%2CNatural%2Cstring

Comment (0atman) [reply src/inim.nim:263]: Otherwise looks good!

**Comment by 0atman** [comment]
` ... /home/oatman/projects/INim/src/inim.nim(260, 7) Error: invalid indentation`

when building this

File: src/inim.nim [hunk 2]
```diff
@@ -258,6 +257,18 @@ proc doRepl() =
     if currentExpression in ["exit", "exit()", "quit", "quit()"]:
         cleanExit()
 
+      if currentExpression in ["help", "help()"]:
```
Comment (0atman) [inline src/inim.nim:260]: Here's the badly-indented code. 

`nimble build && git commit` :laughing:

Comment (Tangdongle) [reply src/inim.nim:260]: Whoops, buggered up the merge. Will fix

**Comment by Tangdongle** [comment]
Aight, fixed up that indentation issue

**Comment by 0atman** [comment]
You sure have!

```
nim> echo "hi"
hi
nim> 1 + 1
2
nim> help
iNim - Interactive Nim Shell - By AndreiRegiani

Available Commands:
Quit - exit, exit(), quit, quit(), ctrl+d
Help - help, help()
```

**Comment by 0atman** [comment]
@Tangdongle Thanks so much for this, really great feature that we can build on :ok_hand:

**Pull request merged**